NAM/dsp.h

```diff
@@ -107,7 +100,7 @@ class DSP
   virtual void _process_core_();
 
   // Copy this->_core_dsp_output to output and apply the output volume
```

Comment:
Nit: Since we're getting rid of `gain`, might be able to simplify this even more.